README.md

```diff
@@ -84,6 +84,7 @@ What kind of metadata can you convey using badges?
 * [ruby-gem-downloads-badge](https://github.com/bogdanRada/ruby-gem-downloads-badge/)
 * [Scrutinizer](https://scrutinizer-ci.com/)
 * [Semaphore](https://semaphoreapp.com)
+* [Sourcegraph](https://sourcegraph.com)
```

Comment:
This is not a list of integrations that the website supports. It is a list of third-parties that attempt to have an implementation conforming to the standard look. I believe that Sourcegraph does not have its own implementation of badges. If that is correct, could you remove this line?